tests/test_physical_channel_property.py

```diff
@@ -0,0 +1,65 @@
+"Contains a collection of pytest tests that validates the physical channel properties."
+import os
+
+from nidaqmx.constants import TerminalConfiguration
+from nidaqmx.tests.test_read_write import TestDAQmxIOBase
+
+
+class TestPhysicalChannelProperty(TestDAQmxIOBase):
+    """Contains a collection of pytest tests that validates the physical channel properties."""
+
+    def test_bool_property(self, any_x_series_device):
+        """Test for validating boolean attributes in physical channel."""
+        phys_chans = any_x_series_device.di_lines
+        assert phys_chans[0].di_change_detect_supported
```

Comment:
@bkeryan  does this value remain constant for all the device ? If not, can we set it from `ini` file?

Reply:
I think all X Series devices support change detection on all DIO lines, so this is fine.